contestreminder/users/views.py

```diff
@@ -3,15 +3,33 @@
 
 # Create your views here.
 from django.urls import reverse_lazy
-from django.views.generic.edit import CreateView
+from django.views.generic.edit import CreateView, UpdateView, DeleteView
 
-from .forms import CustomUserCreationForm
+from .forms import CustomUserCreationForm, CustomUserChangeForm
+from .models import CustomUser
 
 class SignUpView(CreateView):
     form_class = CustomUserCreationForm
     success_url = reverse_lazy('login')
     template_name = 'registration/signup.html'
 
+class ProfileView(UpdateView):
+    model = CustomUser
+    form_class = CustomUserChangeForm
+    success_url = reverse_lazy('profile')
+    template_name = 'user/profile.html'
+
+    def get_object(self):
+        return self.request.user
```

Comment:
1) specify fields as
fields= ['codeChef', 'codeForces', 'hackerEarth', 'hackerRank', 'spoj']

As we don't want user to see password field in update view for now.

Reply:
Done